support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -392,65 +387,6 @@ export function ThreeTierLanding({
 		dispatch(setBillingPeriod(billingFrequencies[buttonIndex]));
 	};
 
-	const handleLinkCtaClick = (
-		event: React.MouseEvent<HTMLAnchorElement>,
-		link: string,
-		price: number,
-		cardTier: 1 | 2 | 3,
-		contributionType: ContributionType,
-		contributionCurrency: IsoCurrency,
-	) => {
-		/** This is a workaround for now while we move tier 3 to a new SupporterPlus ratePlan */
-		if (cardTier === 3) {
-			sendEventContributionCartValue(
-				price.toString(),
-				contributionType,
-				contributionCurrency,
-			);
-			/**
-			 * Lower & middle tier track component click fired via redux side effects.
-			 * Top tier accessed via network request to GuardianWeekly landing page
-			 * therefore tracking required
-			 **/
-			trackComponentClick(
-				`npf-contribution-amount-toggle-${countryGroupId}-${contributionType}-${price}`,
-			);
```

Comment:
@GHaberis should this be fired on the pageview of the checkout?

Reply:
@jamesgorrie `sendEventContributionCartValue` reports to QM the "cart value" so they can attribute financial impacts of issues they pick-up, it should therefore be fired on the pageview of the checkout.

To be honest, I'd look at whether this is easily implementable and if not proceed this PR without implementing this and create a card to revisit `sendEventContributionCartValue`  as it's not a piece of data we're using - so is not a priority.

Reply:
I'd agree that we should should look at writing these methods with the new product model.